src/io/loadActions.ts

```diff
@@ -26,7 +26,9 @@ export const loadActions = (): {
 
   const inputStepsFilePath = process.argv[2];
   const actions: Array<IAction> = [];
-  const currentWorkingDirectory = process.cwd();
+  // copilot finished the next line for me:
+  // look at me - i am the fully retarded javascript language that can't even get the current working directory without a regex
```

Comment:
isn't there something about the number of swear words in a code base is directly related to the quality? i.e. more swears = better quality